### PR TITLE
Remove Measure ID

### PR DESCRIPTION
**Pull request by hanahmily**
Remove the Measure ID constant and Add the Element class to wrap the measure querying response.

Related to https://github.com/apache/skywalking/issues/10105

### CODE REVIEW

File: src/main/java/org/apache/skywalking/banyandb/v1/client/Element.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+/**
+ * RowEntity represents an entity of BanyanDB entity.
+ */
+@Getter
+public class Element extends RowEntity {
+    /**
+     * identity of the element.
+     * For a trace entity, it is the spanID of a Span or the segmentId of a segment in Skywalking,
+     */
+    protected String id;
```
Comment (lujiajing1126) [inline src/main/java/org/apache/skywalking/banyandb/v1/client/Element.java:33]: final?

File: src/main/java/org/apache/skywalking/banyandb/v1/client/MeasureWrite.java [hunk 2]
```diff
@@ -52,10 +51,6 @@ public MeasureWrite tag(String tagName, Serializable<BanyandbModel.TagValue> tag
         return (MeasureWrite) super.tag(tagName, tagValue);
     }
 
-    public MeasureWrite setID(String idValue) throws BanyanDBException {
-        return this.tag(Measure.ID, TagAndValue.idTagValue(idValue));
```
Comment (lujiajing1126) [inline src/main/java/org/apache/skywalking/banyandb/v1/client/MeasureWrite.java:56]: Shall we remove ID type completely?

Comment (hanahmily) [reply src/main/java/org/apache/skywalking/banyandb/v1/client/MeasureWrite.java:56]: Make sense. The ID type means nothing anymore.

File: src/main/java/org/apache/skywalking/banyandb/v1/client/Element.java (hunk 1 above)
Comment (hanahmily) [reply src/main/java/org/apache/skywalking/banyandb/v1/client/Element.java]: done

**Comment by hanahmily** [comment]
I will remove the `ID` type from the server later.

**Comment by hanahmily** [comment]
> > I will remove the `ID` type from the server later.
> 
> So, both Measure and Stream don't have `ID` anymore?

The stream still has an ID https://github.com/apache/skywalking-banyandb-java-client/pull/30/files#diff-b0b913b217ebcd4d170c659a921b48759fd13b9b4e7c738f5679fe25a62000a6R33.

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**